src/main/java/seedu/eduke8/command/AnswerCommand.java

```diff
@@ -0,0 +1,26 @@
+package seedu.eduke8.command;
+
+import seedu.eduke8.common.DisplayableList;
+import seedu.eduke8.option.Option;
+import seedu.eduke8.question.Question;
+import seedu.eduke8.ui.Ui;
+
+public class AnswerCommand extends Command {
+    private Option option;
+    private Question question;
+
+    public AnswerCommand(Option option, Question question) {
+        this.option = option;
+        this.question = question;
+    }
+
+    @Override
+    public void execute(DisplayableList optionList, Ui ui) {
+        if (option.isCorrectAnswer()) {
+            ui.printAnswerIsCorrect();
+            question.markAsAnsweredCorrectly();
+        } else {
+            ui.printAnswerIsWrong();
```

Comment:
I understand that you have incorporated question.markAsShown() function into the getDescription method of question now, however I feel that we should separate them out to enhance SoC and cohesiveness for the method. Will be doing that and hence for this part over here, the questions will have to be marked as shown separately.